### ISSUE TITLE
Page setup instructions throwing 404 error

### ISSUE THREAD

**Issue by wzard** [greglobinski/gatsby-starter-personal-blog#49]
Hi Greg, 

The setup instructions page at **https://dev.greglobinski.com/install-personal-blog-starter/** is showing 404 error. Can you please check. 

Also I hosted a fork of the blog on Gitlab, but the routing and the images are not working. 

![image](https://user-images.githubusercontent.com/23156035/43246174-6dbda216-90ce-11e8-8e35-0f936292b32f.png)

**Comment by greglobinski** [comment]
Thank you @wzard 

It was just a wrong link in the README, it's fixed. It should be https://dev.greglobinski.com/gatsby-starter-personal-blog/

Have you read this https://www.gatsbyjs.org/docs/deploy-gatsby/#gitlab-pages ?

if so

There is a bug in the starter connected with that and  a PR https://github.com/greglobinski/gatsby-starter-personal-blog/pull/45 sent with intention to fix it. Regrettably the PR did not pass the deploy preview so the bug still exists in the code. 

In short you have to move pathPrefix to the root level of gatsby-config object. Now it's in the `siteMetadata` property, move it one level up. 

I hope it'll help.

**Comment by wzard** [comment]
Ah. Perfectly Working.

I changed the pathPrefix in the Gatsby config and provided the "/blog" config in the meta-config. 
Everything is working, now. Thanks Greg.
